src/pytorch_lightning/strategies/launchers/multiprocessing.py

```diff
@@ -31,7 +31,7 @@
 from pytorch_lightning.utilities.types import _PATH
 
 
-class _SpawnLauncher(_Launcher):
+class _MultiProcessingLauncher(_Launcher):
     r"""Spawns processes that run a given function in parallel, and joins them all at the end.
```

Comment:
```suggestion
    r"""Launch processes that run a given function in parallel, and joins them all at the end.
```
or something better?

Reply:
Sure, it wouldn't matter though because here the word is just the plain English word for expressing that processes get created.